packages/blocks/cloudfix/src/index.ts

```diff
@@ -25,7 +25,7 @@ export const cloudfix = createBlock({
       baseUrl: (auth: any) => auth.apiUrl,
       auth: cloudfixAuth,
       authMapping: async (context) => ({
-        Bearer: (context.auth as any).apiKey,
+        Authorization: 'Bearer ' + (context.auth as any).apiKey,
```

Comment:
FYI I used Bearer, because that's also what we use in makeRequest, so we should be consistent https://github.com/openops-cloud/openops/blob/5060acede21128a1c41809728f96c0d9f0a91494/packages/blocks/cloudfix/src/lib/common/make-request.ts#L31